ERCS/erc-7694.md

```diff
@@ -0,0 +1,144 @@
+
+| EVM             | SVM                |
+|:---------------:|:------------------:|
+| `uint8`         | `u8`               |
+| `uint16`        | `u16`              |
+| `uint32`        | `u16`              |
+| `uint64`        | `u64`              |
+| `uint128`       | `u128`             |
+| `uint256`       | `u256`†            |
+| `bytes1`        | `bytes: [u8; 1]`   |
+| `bytes2`        | `bytes: [u8; 2]`   |
+| `bytes4`        | `bytes: [u8; 4]`   |
+| `bytes8`        | `bytes: [u8; 8]`   |
+| `bytes16`       | `bytes: [u8; 16]`  |
+| `bytes32`       | `PubKey`           |
+| `bytes`         | `bytes: Vec<u8>`   |
+| `string`        | `String`           |
+| `address`       | `bytes: [u8; 20]`  |
+
+> † `u256` is not available natively in SVM but is routinely implemented via `u256` crate in Rust
+
+Using this strategy, most - if not all - current use-cases of `StorageRoutedToSolana()` are accounted for.
+
+## Rationale
+`StorageRoutedToSolana()` works in a similar fashion to `StorageRoutedToL2()` in CCIP-Store in the sense that the client needs to be pointed to a certain contract on another chain by the revert event. Other than that, the only technical difference is casting between EVM and SVM data types.
+
+```
+|--------|                                           |-------------------|  |----------------|
+| Client |                                           | Ethereum Contract |  | Solana Program |
+|─-------|                                           |-------------------|  |----------------|
+   |                                                           |                    |       
+   | setValue(calldata)                                        |                    |       
+   |>>---------------------------->-------------------------->>|                    |       
+   |                                                           |                    |       
+   | revert StorageRoutedToSolana(programId, account)          |                    |       
+   |<<----------------------------<--------------------------<<|                    |       
+   |                                                           |                    |       
+   | Execute on Solana [programId] [account] [callData]        |                    |       
+   |>>---------------------------->---------------------------->---------->------->>|      
+   |                                                           |                    |       
+   | response                                                  |                    |       
+   |<<----------------------------<----------------------------<----------<-------<<|
+   |                                                           |                    |       
+```
+
+## Backwards Compatibility
+None.
+
+## Security Considerations
+None.
+
+## Copyright
+Copyright and related rights waived via [`CC0`](../LICENSE.md).
```

Comment:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE.md).
```

Reply:
Fixed in https://github.com/ethereum/ERCs/pull/390/commits/a57b5457833621a5800abaec9d214fb552b1b1a4